### ISSUE TITLE
Regarding Training & Prediction

### ISSUE THREAD

**Issue by ghost** [vvuonghn/AI_DocumentLayoutAnalysis#1]
@hongocvuong1998 Thank you for your hard work.

**General:**
- What are the required dependencies & versions to run this program?
Example: (Opencv 3 or 2?  Python 3 or 2.7 ?, etc..)
- Why didn't you use [Conda](www.anaconda.com/download) instead of Visual Studio?
(Conda can run on multiple platforms: Windows, Linux, Mac)

**Training:**
- What is required for training?
- How to create our own xml labels?
- Can you provide some ground-truth data to test training?
- Can I train from scratch? or I must use the pre-trained `vgg16.npy` model?
- Can your program be trained to detect Text-Lines? or does it only detect Paragraph regions?

**Prediction:**
- How can I predict using the model I trained?
- Can I extract the images of the predicted regions?

Waiting for your reply

**Comment by hongocvuong1998** [comment]
Can you contact with me, I will explain in detail. 
mail: hongocvuong1998@gmail.com

**Comment by vvuonghn** [comment]
> @hongocvuong1998 Thank you for your hard work.
> 
> **General:**
> 
> * What are the required dependencies & versions to run this program?
>   Example: (Opencv 3 or 2?  Python 3 or 2.7 ?, etc..)
> * Why didn't you use [Conda](www.anaconda.com/download) instead of Visual Studio?
>   (Conda can run on multiple platforms: Windows, Linux, Mac)
> 
> **Training:**
> 
> * What is required for training?
> * How to create our own xml labels?
> * Can you provide some ground-truth data to test training?
> * Can I train from scratch? or I must use the pre-trained `vgg16.npy` model?
> * Can your program be trained to detect Text-Lines? or does it only detect Paragraph regions?
> 
> **Prediction:**
> 
> * How can I predict using the model I trained?
> * Can I extract the images of the predicted regions?
> 
> Waiting for your reply

Can you contact with me, I will explain in detail.
mail: hongocvuong1998@gmail.com
Thank you for care about my project
